MAINTAINERS.rst

```diff
@@ -88,7 +88,8 @@ Building and uploading the release wheels
 You need the zephyr-project PyPI credentials for the 'twine upload' command. ::
 
   git clean -ffdx
-  python3 setup.py sdist bdist_wheel
+  python3 -m pip install --upgrade build twine
```

Comment:
- Why `python3 -m pip install...` instead of the usual, shorter and clearer `pip3 install ...`?

- Rather than blindly upgrading "the latest", how about adding `build>=1.something` to `[dependencies]` in `pyproject.toml`?
`twine` is different because it is not needed for local installations.

So I think this script would be more consistent with the English text:

```
git clean -fdx
python3 -m build
pip3 install --upgrade twine
twine upload ...
```

Reply:
`pyproject-build` is also not needed for local installations. I can do `pip install .` without having `pyproject-build` installed.

Reply:
Indeed, thanks for correcting me. It's not "built-in" either: it just happened to be already installed on my system for some reason.